Client/Frontend/Widgets/SnackBar.swift

```diff
@@ -167,12 +172,9 @@ class SnackBar: UIView {
         return true
     }
 
-    override func updateConstraints() {
-        super.updateConstraints()
-
+    private func setupButtonViewLayout(constant: CGFloat = 0) {
         buttonsView.snp.remakeConstraints { make in
-            make.top.equalTo(titleView.snp.bottom).offset(UIConstants.DefaultPadding)
-            make.bottom.equalTo(self.snp.bottom)
+            make.bottom.equalTo(self.snp.bottom).inset(constant)
```

Comment:
```suggestion
            make.bottom.equalTo(self.safeAreaLayoutGuide.snp.bottom)
```

Reply:
If we constraint the bottom to the safeAreaLayoutGuide we don't need to set a fixed height inset. It works on every device and so there is no need anymore to check for iPad.